ansible/roles/vm_set/tasks/main.yml

```diff
@@ -148,6 +148,19 @@
     sysctl_set: yes
   become: yes
 
+- name: Create link for disabling apparmor for ntpd
+  file:
+    src: /etc/apparmor.d/usr.sbin.ntpd
+    dest: /etc/apparmor.d/disable/usr.sbin.ntpd
+    state: link
+  become: yes
+  ignore_errors: yes
+
+- name: Disable apparmor for ntpd
```

Comment:
shouldn't this task make use of the result of the previous task? what happens if you just run this command without the link in place?

Reply:
The apparmor_parser command is required to disable apparmor for a service.
https://www.cyberciti.biz/faq/ubuntu-linux-howto-disable-apparmor-commands/

Reply:
how do we ensure that apparmor is disabled for ntp since we seem to ignore errors in both the task?

Reply:
Most of the test setups have apparmor disabled for NTP. In this case running the tasks would fail. These are just two best effort tasks. So, 'ignore_errors' is required.